contracts/sweep/TokenSweep.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+/**
+ * @title Abstract upgradeable contract that provides the ability to sweep erroneous transferred tokens to a defined beneficiary address
+ *
+ * @dev Access control implementation is required for many functions by design
+ */
+abstract contract TokenSweep {
+    address internal _beneficiary;
+
+    event BeneficiaryUpdated(address indexed beneficiary);
+
+    function beneficiary() external view returns (address) {
+        return _beneficiary;
+    }
+
+    /**
+     * @notice Set the beneficiary of the token sweep
+     *
+     * @dev Needs access control implemented in the inheriting contract
+     * @param newBeneficiary The address of the beneficiary
+     */
+    function _setBeneficiary(address newBeneficiary) internal {
+        require(newBeneficiary != address(0), "TokenSweep: beneficiary-zero");
+        require(newBeneficiary != address(this), "TokenSweep: self-address");
+        require(newBeneficiary != _beneficiary, "TokenSweep: not-updating");
```

Comment:
nit; message does really convey the problem.

This requirement being the address is not the same as the beneficiary already set ....perhaps something along the lines of `TokenSweep: beneficiary identical`?

Reply:
"TokenSweep: beneficiary-identical" is getting an error message too long error!

Reply:
removing the space after the colon to "TokenSweep:beneficiary-identical" works but is inconsistent?